data_structures/src/chain.rs

```diff
@@ -4989,4 +5052,572 @@ mod tests {
         let expected_order = vec![pkh_b, pkh_a];
         assert_eq!(rep_engine.get_rep_ordered_ars_list(), expected_order);
     }
+
+    #[test]
+    fn test_dr_merkle_root_superblock() {
+        let inputs: Vec<Input> = (1..4)
+            .map(|x| {
+                Input::new(OutputPointer {
+                    transaction_id: Hash::default(),
+                    output_index: x,
+                })
+            })
+            .collect();
+        let dr_txs: Vec<DRTransaction> = inputs
```

Comment:
I think this is repeated in all tests, how about creating a function like `example_drs(n)` that returns n transactions